sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.controller.service.ts

```diff
@@ -84,7 +84,7 @@ export class DesignationAgreementControllerService {
           submittedDate: designation.submittedDate,
           startDate: getISODateOnlyString(designation.startDate),
           endDate: getISODateOnlyString(designation.endDate),
-        } as GetDesignationAgreementsDto),
+        } as DesignationAgreementDetailsAPIOutDTO),
```

Comment:
We try to avoid the cast in the DTO return to avoid sending unexpected default values.